doc/adrs/010-glossary.adoc

```diff
@@ -0,0 +1,40 @@
+= ADR-010 - Adopt a Shared Vocabulary
+
+== Context
+
+Given our long history with Sirius Desktop and the way it used modeling and EMF-specific terms, we tend to reuse these (at least in our discussions).
+For Sirius Web, we do not want to be limited to EMF, or even a modeling/MDE point of view, se we need to use different terms.
+
+We need to decide on a common vocabulary, both for the user-visible elements (e.g. do we talk about "Documents", or "Models"? is it "Modeler" or "Studio", or do both exist but represent slightly different things?) and also more internally on the technical terms.
+
+== Decision
+
+Roles:
+* end user
+    * viewer
+    * contributor
+* specifier
+* project admin
+* server/instance admin
+
+Concepts
+* Organisation
+* Project
+* Document (model instance)
```

Comment:
Where document would be used ?

Reply:
Supposedly in place of "Model", as we wanted to avoid this (loaded) term, and "Document" is what is used internally.
But "document" feels too generic and without context evokes something like a Word/text-based document (at least to me).
I'll switch to "Model" for now, and try to propose other terms.